KoreBuild-dotnet/build/dotnet-install.ps1

```diff
@@ -59,6 +59,8 @@ if (Test-Path $LocalFile)
                 say "You already have the latest version"
                 exit 0
             }
+            $DotNetFileName="dotnet-win-x64.$RemoteVersion.zip"
```

Comment:
The same bug was present in the powershell script, as in the bash script. This just brings both scripts to functional parity.